apps/drec-api/src/pods/user/dto/create-user.dto.ts

```diff
@@ -14,6 +20,11 @@ export class CreateUserDTO
   implements UserRegistrationData
 {
   @ApiProperty({ type: String })
+  @MinLength(6)
+  @MaxLength(20)
+  @Matches(/((?=.*[0-9])(?=.*[a-z]).{6,})/, {
+    message: 'password must contain at least one digit',
```

Comment:
technically this message doesn't reflect regexp requirement

Reply:
Well it could also say that it requires min 6 characters but this message is also coming from @MinLength so I thought it would be redundant. I can remove MinLength and then change the message.

Reply:
And it also requires at least one letter.

Reply:
And the letter cannot be latin-extended, only basic a-z. Also it cannot be big, because then it won't work

Reply:
I updated the message